packages/generator/tests/test_assets/example_renders/measurement_plugin_client/non_streaming_data_measurement_client.py

```diff
@@ -414,11 +414,12 @@ def _create_measure_request(
         self, parameter_values: List[Any]
     ) -> v2_measurement_service_pb2.MeasureRequest:
         serialized_configuration = any_pb2.Any(
+            type_url="type.googleapis.com/ni.measurementlink.measurement.v2.MeasurementConfigurations",
```

Comment:
Sigh, this highlights something else that's wrong.

https://github.com/ni/measurement-plugin-python/issues/890